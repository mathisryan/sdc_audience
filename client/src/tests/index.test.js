import React from 'react';
import { shallow } from 'enzyme';
import App from '../index.jsx';
import './setupTest'

describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<App debug />);
    expect(component).toMatchSnapshot();
  });
});