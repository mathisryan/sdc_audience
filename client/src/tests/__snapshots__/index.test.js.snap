// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `
<div>
  <AudienceReview
    reviews={Array []}
    title=""
  />
</div>
`;
